main.go

```diff
@@ -7,7 +7,14 @@ import (
 	"github.com/giantswarm/e2e-harness/cmd"
 )
 
+var (
+	gitCommit = "latest"
```

Comment:
We usually set it to `n/a`. https://github.com/giantswarm/kvm-operator/blob/328897482d9d15543dd4c50ea2449628a3cc316b/main.go#L23